### PR TITLE
21878  - AUTH WEB - Mailing address not update when switch accounts

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/21878

*Description of changes:*
1. fix for: Mailing address not update when switch accounts
2. update relate components BaseAddressForm.vue and AccountMailingAddress.vue to composition-api

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/common/BaseAddressForm.vue [hunk 1]
```diff
@@ -11,52 +11,79 @@
...
-    // convert to address format to component
-    this.inputaddress = CommonUtils.convertAddressForComponent(this.address)
-    this.emitUpdateAddress(this.inputaddress)
+    function watchEditing (editing: boolean) {
+      if (!editing) {
+        loadAddressIntoInputAddress()
+      }
+    }
+
+    // Watch the editing prop
```
Comment (seeker25) [inline auth-web/src/components/auth/common/BaseAddressForm.vue:65]: redundant

File: auth-web/src/components/auth/common/BaseAddressForm.vue [hunk 2]
```diff
@@ -11,52 +11,79 @@
...
+    watch(() => props.editing, watchEditing)
+
+    // Watch the address prop
```
Comment (seeker25) [inline auth-web/src/components/auth/common/BaseAddressForm.vue:68]: redundant

**Review by seeker25 — APPROVED** [review]
LGTM

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2874-kely4b9d.web.app

**Pull request merged**